Parser/asdl_c.py

```diff
@@ -391,9 +395,8 @@ def sumTrailer(self, name, add_label=False):
     def simpleSum(self, sum, name):
         self.funcHeader(name)
         for t in sum.types:
-            line = ("isinstance = PyObject_IsInstance(obj, "
-                    "(PyObject *)%s_type);")
-            self.emit(line % (t.name,), 1)
+            self.emit("isinstance = PyObject_IsInstance(obj, "
+                      f"astmodulestate_global->{t.name}_type);", 1)
```

Comment:
You cannot use f-strings here. Some platforms like old Ubuntu have python3 3.5 or older. Travis CI is failing with a SyntaxError.